service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/InitiatedCaseEventHandlerTest.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+public class InitiatedCaseEventHandlerTest {
```

Comment:
I guess we could remove that test as there is very small responsibility in handler itself.